warehouse/query-core/src/test/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTermsTest.java

```diff
@@ -490,6 +488,27 @@ public void testDelayedPredicates() throws ParseException {
         }
     }
     
+    @Test
+    public void testFailedRegexNormalizers() throws ParseException {
```

Comment:
Maybe a few more test cases? 
Consider the case when the field is index only.

Reply:
looking at fixing the HitListArithmetic to be consistent with the DefaultArithmetic.